packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -70,13 +70,33 @@ export default function globalReferenceToImport(
   }
 
   function findNamedExports(filename) {
-    if (extname(filename) === '.coffee') {
-      return [];
+    let namedExports = [];
+    switch (filename.slice(filename.indexOf('.'))) {
```

Comment:
`extname` will only give `.erb` for `.js.erb`.

After discussing IRL we realized it should be okay since we only care about if the final extension is `.js` or not so I will go back to `extname`.